content/docs/for-buildpack-authors/tutorials/basic-buildpack/02_building-blocks-cnb.md

```diff
@@ -25,7 +25,7 @@ pack buildpack new examples/node-js \
     --api 0.10 \
     --path node-js-buildpack \
     --version 0.0.1 \
-    --stacks io.buildpacks.samples.stacks.noble
+    --targets "linux/amd64"
```

Comment:
question: While you are here, maybe

```diff
diff --git a/content/docs/for-buildpack-authors/tutorials/basic-buildpack/02_building-blocks-cnb.md b/content/docs/for-buildpack-authors/tutorials/basic-buildpack/02_building-blocks-cnb.md
index 55c72761..6ddb7977 100644
--- a/content/docs/for-buildpack-authors/tutorials/basic-buildpack/02_building-blocks-cnb.md
+++ b/content/docs/for-buildpack-authors/tutorials/basic-buildpack/02_building-blocks-cnb.md
@@ -34,7 +34,7 @@ This command will create `node-js-buildpack` directory which contains `buildpack
 - `-a, --api` Buildpack API compatibility of the generated buildpack
 - `-h, --help` Help for 'new'
 - `--path` the location on the filesystem to generate the artifacts
-- `--stacks` Stacks (deprecated) the buildpack will work with
+- `--targets` OS and arch the buildpack will target
 - `-V, --version` the version of the buildpack in buildpack.toml
```

or similar makes sense?

Reply:
I've bumped this in https://github.com/buildpacks/docs/pull/793.  We will update the text when your changes to `pack --targets` lands.  https://github.com/buildpacks/pack/pull/2296